### PR TITLE
Fix/Align breadcrumb style for service

### PR DESCRIPTION
**Pull request by WilliamTraoreee**
# What does this PR do?

> [Link to the JIRA ticket](https://qovery.atlassian.net/browse/FRT-439?atlOrigin=eyJpIjoiMDQzMzNiOGE1MDBkNDNjNDk3NmRjM2Q4ZGUwYThkYWEiLCJwIjoiaiJ9)

If you are on a service, the breadcrumb doesn't add the correct style and on click redirect to the service list

image

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - Fix/Align breadcrumb style for service - 2022-06-24T08:21:26Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/e6a375cf-f022-4b80-9e1e-6d5ea9f62743/applications)

**Comment by bdebon** [comment]
gg

File: libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.tsx [hunk 1]
```diff
@@ -106,32 +108,46 @@ export function Breadcrumb(props: BreadcrumbProps) {
...
-            contentLeft: (
-              <div className="flex items-center">
-                <StatusChip status={application.status?.state} />
-                <div className="ml-3 mt-[1px]">
-                  <Icon
-                    name={!(application as ApplicationEntity).build_mode ? IconEnum.APPLICATION : IconEnum.DATABASE}
-                    width="16"
-                  />
+        ? [
+            ...(applications?.map((application: ApplicationEntity) => ({
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.tsx:115]: Can we have only one map for this? it's the same logic just one Icon and the link who changes?

**Review by RemiBonnet — COMMENTED** [review]
Well done, thanks @WilliamTraoreee !
I have just one suggestion to improve the code

**Pull request merged**